deploy/docker/entrypoint.sh

```diff
@@ -1,19 +1,20 @@
 #!/bin/sh
-if [ $# -ne 3 ]
+if [ $# -ne 2 ]
 then
   echo "Illegal number of parameters. Exiting..."
-  echo "Command: ./entrypoint.sh \${mongo_host} \${mongo_port} \${shared_pvc}"
+  echo "Command: ./entrypoint.sh \${mongo_host} \${mongo_port}"
```

Comment:
This changes a bit the principle that was in place where:
- mandatory values are entrypoint arguments
- optional values are environment variables